systests/databinding/pom.xml

```diff
@@ -165,7 +165,7 @@
             <plugin>
                 <groupId>org.jibx</groupId>
                 <artifactId>maven-jibx-plugin</artifactId>
-                <version>${cxf.jibx.version}</version>
+                <version>1.3.1</version>
```

Comment:
Can this PR be closed? We can't apply the fix as otherwise the branch won't build with older JDK versions.